### ISSUE TITLE
predict.py does not differentiate apnea and non-apnea arousals

### ISSUE THREAD

**Issue by Jhanyi** [GuanLab/DeepSleep#5]
Hello,

thanks very much for open-sourcing your code :) my question is regarding the mention in your accompanying paper in communications biology: "We have therefore built neural network models for detecting apnea, in addition to the model for detecting non-apneic arousals ... Specifically we applied DeepSleep pipeline to the PhysioNet data and built three types of models for detecting (1) apneic arousals; (2) non-apenic arousals; ..."

Running predict.py on tr03-0078 and tr03-0079 seems to predict both apnea and non-apnea. Is the model that distinguish them included in this repo? Thank you

**Comment by Hongyang449** [comment]
Hi @Jhanyi ,

The pre-trained model weights in this repo is the version from the 2018 PhysioNet challenge, where we predicted non-apneic arousals (labeled by 1) VS others (labeled by 0). See the objective of the challenge here:
https://physionet.org/content/challenge-2018/1.0.0/

If you need a distinguishable model, you can adapt the pipeline by changing the labels based on apneic VS non-apneic , then re-train the model.

Thanks,
Hongyang

**Comment by Jhanyi** [comment]
I see, I thought it was included in the repo. Thanks @Hongyang449 for clarifying.
